README.md

```diff
@@ -5,3 +5,7 @@ Using Ultralytics YOLOv8.
 ## Training
 
 Follow the [instructions](https://uwarg-docs.atlassian.net/l/cp/XBXNaCnx).
+
+## Imaging
+
+Follow the [instructions](https://uwarg-docs.atlassian.net/l/cp/JuBLUpJr).
```

Comment:
Are you planning on making another PR adding to the `README` with instructions on setup?

Reply:
That Confluence page will contain instructions. I will hold off on merging until they are written.